boilerplates/react-union-boilerplate-liferay-basic/tools/amdLoaderScripts.js

```diff
@@ -26,12 +26,12 @@ const targetDirectory = path.join(
 	appDirectory,
 	'liferay/amd-loader/src/main/resources/META-INF/resources'
 );
-const configFilePath = path.join(targetDirectory, 'config.js');
+const configFilePath = path.join(buildDirectory, 'config.js');
 
 const joinNonEmpty = xs => xs.filter(Boolean).join('.');
 
 const getLiferayFilePath = (appName, hash) =>
-	path.join(targetDirectory, `${joinNonEmpty([appName, hash])}.js`);
```

Comment:
You can remove the targetDirectory variable altogether, it's not used anywhere else :)

Reply:
Done